account_bank_statement_import_qif/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,14 @@
+This module allows you to import the machine readable QIF Files in Odoo: they
+are parsed and stored in human readable format in
+Accounting \ Bank and Cash \ Bank Statements.
+
+Important Note
+--------------
```

Comment:
Please use `~~~~~~`, as it's the subtitle indicator accepted as this level of the RST when composing the whole README.